lib/engine/game/g_1860.rb

```diff
@@ -6,6 +6,8 @@
 module Engine
   module Game
     class G1860 < Base
+      attr_reader :cobank
```

Comment:
what's a cobank

Reply:
Company bank.

Reply:
I'm open to suggestions for a better name. The idea is to have two banks: the player bank (that breaks), and a company bank used for funds into and out of corporations. Most of the time (except for 1860 now) the company bank is just an alias for the player bank.

Reply:
is it just an infinite pool of money?

Reply:
Yes

Reply:
ok rename it something like that instead of cobank

Reply:
But for most games the corporation is transacting with a finite bank, so naming it "infinite_pool" wouldn't make sense either. What would you suggest? Maybe "corporate_bank"?

Reply:
this is an 1860 only construct, so corporate_bank or infinite_bank is fine.

Reply:
But I'm trying to refactor common code to use either the player bank or the corporate bank. Otherwise, I wind up doing ugly things like creating duplicates of a Steps just to change the bank in the spend method.